rpcs3/Emu/Cell/SPULLVMRecompiler.cpp

```diff
@@ -3532,6 +3551,50 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 		}
 		}
 
+		if (m_inst_attrs[(m_pos - m_base)  / 4] == inst_attr::rchcnt_loop)
```

Comment:
```suggestion
		if (m_inst_attrs[(m_pos - m_base) / 4] == inst_attr::rchcnt_loop)
```

Reply:
is there a specific keyword (e.g. RCHCNT) traced in the log file that we can search for to understand if the game is affected by this improvement?